### PR TITLE
Added MPS Youtube support

### PR DESCRIPTION
**Pull request by fmartingr**
- Sync app configuration file
- Sync app local playlists file

### CODE REVIEW

**Comment by fmartingr** [comment]
Solved conflicts.

File: mackup/applications/mps-youtube.cfg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[application]
+name = MPS Youtube
+
+[configuration_files]
+.config/mps-youtube/config
+.config/mps-youtube/playlist_v2
```
Comment (vitorgalvao) [inline mackup/applications/mps-youtube.cfg]: It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

Comment (fmartingr) [inline mackup/applications/mps-youtube.cfg]: I can agree with you but I've seen some other config files without it so I'm leaving it as it is to match the current config files.

Comment (vitorgalvao) [inline mackup/applications/mps-youtube.cfg]: > but I've seen some other config files without it

Those are also wrong.

> I'm leaving it as it is to match the current config files.

That makes no sense. Some config files have this error, others do not, and you’re choosing to follow the objectively wrong behaviour.

The files that have this error should all be fixed.

Comment (fmartingr) [inline mackup/applications/mps-youtube.cfg]: It's not a big deal IMO, but I want to know what @lra have to say about it, because I prefer to do a separate PR adding all the obviously needed line breaks rather than just *doing it right* for this file only.

Comment (lra) [inline mackup/applications/mps-youtube.cfg]: > I've seen some other config files without it so I'm leaving it as it is to match the current config files.

Not sure what you refer to. All the cfg files end with a newline in this repo. @vitorgalvao's link explains why it's important.

> but I want to know what @lra have to say about it

Just thanks @vitorgalvao for catching this.

Comment (fmartingr) [inline mackup/applications/mps-youtube.cfg]: I'm sorry for my behaviour yesterday, I had a really bad day and acted like an idiot.
I've fixed the issue now, thanks @vitorgalvao for pointing it out.

**Comment by lra** [comment]
thx

File: mackup/applications/mps-youtube.cfg (hunk 1 above)
Comment (vitorgalvao) [inline mackup/applications/mps-youtube.cfg]: > I'm sorry for my behaviour yesterday, I had a really bad day and acted like an idiot.

No worries here, it’s all good. We disagreed, but it’s not like I thought you were being rude, or anything. The conversation remained civil, and that’s what matters.
